content/departments/engineering/teams/cody/cody-gateway/using.md

```diff
@@ -21,7 +21,10 @@ Access to the production Cody Gateway instance can be provisioned with the follo
    1. If you have the license key on hand, you can also use the [license key lookup tool](https://sourcegraph.com/site-admin/dotcom/product/licenses).
 3. Under "Cody services":
    1. Enable access to Cody Gateway
-   2. If desired, configure a custom rate limit for the desired features
+   2. If desired, configure custom rate limits to allow more/less usage
+   3. **Customers can only use the models listed under "Cody services"**. To configure other models:
+      1. Adding the `gpt` tag to the latest license will enable access to all OpenAI GPT models.
+      2. Adding a custom rate limit allows you to configure other models to allow the customer to use.
```

Comment:
Should we also mention or is it necessary to update site config to explicitly use OpenAI GPT models? Or is the `gpt` tag in the license key automatically changing the model that is used?

Reply:
This section is only _provisioning_ access, configuration is a separate step in the next section

Reply:
Oh, got it!

Reply:
Added a line to clarify!